### ISSUE TITLE
"in" keyword is not being recognized

### ISSUE THREAD

**Issue by jameskolce** [hlissner/emacs-pug-mode#6]
<img width="289" alt="screenshot at dic 06 6-30-45 p m" src="https://cloud.githubusercontent.com/assets/2559808/20951299/3c41c55e-bbe2-11e6-9a97-f752740a239b.png">

I might take a look at it though. I wanted to submit the issue just in case.

**Comment by hlissner** [comment]
If it isn't too much trouble, please do. I don't use the logic statements in Pug much, so I'll bet there are issues hiding in there, and my elisp-fu is still a bit young where major modes are concerned.

**Comment by jameskolce** [comment]
Sure, I’m on it right now! I made some changes and it's already working but we have this:

```elisp
;; Clear after : or selectors
("[[:alnum:]_)]\\(?::\\s-+[^ ]+\\|\\s-+\\)\\([^\n]*\\)"
  (beginning-of-line) nil
  (1 nil t)))
```

And it resets the highlighting of everything after a keyword (both Pug keywords and HTML tags), so I have to change it to match stuff only after HTML tags. Still trying to figure out how to do that. I tried with the `pug-tags-re` but it didn't work out. I think I'm close though, so I'll let you know.

I'm basically moving things around, I just started using emacs the last week lol.
